ui/src/actions/ws.ts

```diff
@@ -21,6 +21,9 @@ export const wsConnect =
     const webSocketUrl = (() => {
       const url = new URL("/ws", window.location.href);
       url.protocol = url.protocol.replace("http", "ws");
+      if (process.env.NODE_ENV === "development") {
```

Comment:
Id also add a note here stating why we are doing this.

Reply:
Done!